libs/labelbox/src/labelbox/data/annotation_types/collection.py

```diff
@@ -40,26 +40,6 @@ def _assign_ids(label: Label):
         self._fns["assign_feature_schema_ids"] = _assign_ids
         return self
 
-    def add_url_to_data(
```

Comment:
Removed this function. This is not supported by GenericDataType and I do not think this is relevant any longer. Check doc string for what it does